python/pyspark/ml/torch/utils.py

```diff
@@ -0,0 +1,24 @@
+from pyspark.context import SparkContext
```

Comment:
license header missing

Reply:
I will actually delete this file for now since I don't expect a lot of utils just yet and if we do, we can move it out to `utils.py` at a later time.